applications/nrf_desktop/configuration/nrf52840gmouse_nrf52840/app_ZDebugSplitLL.conf

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 ################################################################################
+# DebugSplitLL version
```

Comment:
I wonder if we should add the SplitLL configurations to the common list in README. They are used for quite a lot of boards.

Reply:
Done